modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -96,6 +64,37 @@ Spark can use the REST catalog to automatically discover the topic's Iceberg tab
 
 Depending on your processing engine, to xref:manage:iceberg/query-iceberg-tables.adoc[query the Iceberg data] you may need to create a new table in your data warehouse or lakehouse. See also: xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[]
 
+== File system-based catalog (`object_storage`)
```

Comment:
Suggest renaming these headings to **Integrate with file system catalog** and **Integrate with REST catalog**.